pkg/pillar/cipher/handlecipher.go

```diff
@@ -169,18 +155,22 @@ func DecryptCipherBlock(ctx *DecryptCipherContext,
 }
 
 func decryptCipherBlockWithECDH(ctx *DecryptCipherContext,
-	cipherBlock types.CipherBlockStatus) ([]byte, error) {
-	cert, err := getControllerCertEcdhKey(ctx, cipherBlock)
+	cipherContext *types.CipherContext, cipherBlock types.CipherBlockStatus) ([]byte, error) {
+	ecdhKey, err := getControllerCertEcdhKey(ctx, cipherContext.ControllerCertKey())
```

Comment:
This function returns a certificate, right? If so why did you rename the local variable to ecdhKey?

Reply:
reverted.